scripts/report-bundle-size/report-as-a-pr-comment.js

```diff
@@ -1,31 +1,26 @@
-const { command } = require('./lib/command')
-const { runMain, timeout } = require('./lib/execution-utils')
-const { getOrg2ApiKey, getGithubAccessToken, getOrg2AppKey } = require('./lib/secrets')
+const { command } = require('../lib/command')
+const { getOrg2ApiKey, getGithubAccessToken, getOrg2AppKey } = require('../lib/secrets')
 
 const PR_COMMENT_HEADER = 'Bundles Sizes Evolution'
-const PACKAGES_NAMES = ['rum', 'logs', 'rum_slim', 'worker']
 const BASE_BRANCH = process.env.MAIN_BRANCH
 const LOCAL_BRANCH = process.env.CI_COMMIT_REF_NAME
 const PR_COMMENTER_AUTH_TOKEN = command`authanywhere`.run().split(' ')[2].trim()
 const GITHUB_TOKEN = getGithubAccessToken()
 const ONE_DAY_IN_SECOND = 24 * 60 * 60
-const TIMEOUT_DURATION_MS = 5000
-const FETCH_RETRIES = 6
 
-runMain(async () => {
+async function reportBundleSizes(localBundleSizes) {
```

Comment:
**💬 suggestion:** ‏make it more explicit so it's clearer when using it in the main script
```suggestion
async function reportBundleSizesAsPrComment(localBundleSizes) {
```